packages/button/theme/lumo/vaadin-button-styles.js

```diff
@@ -0,0 +1,302 @@
+import { registerStyles, css } from '@vaadin/vaadin-themable-mixin/register-styles.js';
+import '@vaadin/vaadin-lumo-styles/color.js';
+import '@vaadin/vaadin-lumo-styles/sizing.js';
+import '@vaadin/vaadin-lumo-styles/spacing.js';
+import '@vaadin/vaadin-lumo-styles/style.js';
+import '@vaadin/vaadin-lumo-styles/typography.js';
+
+registerStyles(
+  'vaadin-button',
```

Comment:
Let's use the same approach as in #2324 and prevent applying new CSS to the old component implementation

You can make it easier by not passing a tag:
```suggestion
registerStyles(
  '',
```

Reply:
Forgot about such a case, thanks! Done.